README.md

```diff
@@ -85,8 +88,8 @@ If you'd like to use these custom ops, you can install from source using the
 instructions below.
 
 ### Installing KerasCV with Custom Ops from Source
-Installing from source requires the [Bazel](https://bazel.build/) build system
-(version >= 5.4.0).
+Installing custom ops from source requires the [Bazel](https://bazel.build/) build
+system (version >= 5.4.0).
```

Comment:
I suggest provide a bazel install instructions here since the official website of Bazel doesn't have an easy guide for installation.

May refer to the following:
https://github.com/keras-team/keras/blob/v2.11.0/.devcontainer/Dockerfile#L21-L23

Reply:
FWIW we also have the Docker install for Bazel here:
https://github.com/keras-team/keras-cv/tree/master/cloudbuild#kerascv-accelerators-testing